src/components/FeatureViews/Accounts/TabComponents/Schedule.js

```diff
@@ -8,10 +10,37 @@ class Schedule extends Component {
         };
     }
 
+    // get instructors schdeule and parses it into one array of objcts 
+    getInstructorSchedule = () => {
+        let schedule = this.props.work_hours
+        let key = Object.keys(this.props.work_hours)
+
+        let instructorsSchedule = key.map((iKey) => {
+            return schedule[iKey]
+        })
+        let finalInstructorSchedule = []
+        instructorsSchedule.forEach((iList) => {
+            finalInstructorSchedule = finalInstructorSchedule.concat(Object.values(iList))
+        })
+        return instructorsSchedule
+    }
 
 
     render() {
-        return(<h1>Schedule</h1>)
+
+        return (
+            <div>
+                <FullCalendar
+                    header={false}
+                    columnHeaderFormat={{ weekday: "short" }}
+                    allDaySlot={false}
+                    height={337}
+                    defaultView="timeGridWeek"
+                    plugins={[timeGridPlugin]}
+                    events={this.getInstructorSchedule()}
+                />
+            </div>
+        )
```

Comment:
Since this entire component is no more than a `render()` function, consider rewriting this as a function component.